test_conformance/select/test_select.c

```diff
@@ -573,8 +575,18 @@ test_definition test_list[] = {
 
 const int test_num = ARRAY_SIZE( test_list );
 
-int main(int argc, char* argv[])
+int main(int argc, const char* argv[])
 {
+    test_start();
+
+    argc = parseCustomParam(argc, argv);
+    if (argc == -1)
+    {
+        test_finish();
```

Comment:
`test_finish` doesn't seem to do much these days but it's all over the place so we might as well keep it here now that you've done the work.